### PR TITLE
Add color picker, alpha support

### LINKED ISSUE #45: Support Color Picker
Could be awesome be possible open a color picker for setup the background.

This also be possible setup `transparent` as brakcgorund.

### PR DESCRIPTION
**Pull request by jakedex**
Closes #45

### CODE REVIEW

**Comment by yeskunall** [comment]
Oo, can't wait for this to land! 👍

File: components/Carbon.js [hunk 1]
```diff
@@ -112,7 +113,38 @@ class Carbon extends React.Component {
             value={this.props.children}
             options={options}
           />
+          <div id="container-bg">
```
Comment (mfix22) [inline components/Carbon.js]: https://www.dropbox.com/s/68ufy8w1d5gtmrz/Screenshot%202017-10-04%2022.38.43.png?dl=0

This pixel is still a little off homie

Comment (jakedex) [reply components/Carbon.js]: Agh, okay. Will fix tonight. Good catch.

**Comment by Kikobeats** [comment]
awesome!!!!!!

**Pull request merged**